package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "keyboard-key",
-  "version": "1.0.4",
+  "version": "1.1.0",
```

Comment:
Oops, the `package.json` `version` gets incremented automatically by the `npm` CLI in an `npm release`. I'll fix before releasing, just FYI.